go.mod

```diff
@@ -1,6 +1,8 @@
 module github.com/k3s-io/k3s
 
-go 1.20
+go 1.21
+
+toolchain go1.21.4
```

Comment:
shouldn't this be `go1.21.5`, as that's the specific golang patch release being used by upstream?

Reply:
Also, if we are going to decouple this from the golang version used in the builder, we will need to update the checks in the validate script to ensure that the actual toolchain being used is the correct version:
https://github.com/k3s-io/k3s/blob/71a3c35fb7077c5a952fbc74a9cd8dee352b1178/scripts/validate#L31-L37